api/estados/saopaulo.js

```diff
@@ -1,652 +1,654 @@
+    'Pedra Bela',
+    'Pedranópolis',
+    'Pedregulho',
+    'Pedreira',
+    'Pedrinhas Paulista',
+    'Pedro de Toledo',
+    'Penápolis',
+    'Pereira Barreto',
+    'Pereiras',
+    'Peruíbe',
+    'Piacatu',
+    'Piedade',
+    'Pilar do Sul',
+    'Pindamonhangaba',
+    'Pindorama',
+    'Pinhalzinho',
+    'Piquerobi',
+    'Piquete',
+    'Piracaia',
+    'Piracicaba',
+    'Piraju',
+    'Pirajuí',
+    'Pirangi',
+    'Pirapora do Bom Jesus',
+    'Pirapozinho',
+    'Pirassununga',
+    'Piratininga',
+    'Pitangueiras',
+    'Planalto',
+    'Platina',
+    'Poá',
+    'Poloni',
+    'Pompéia',
+    'Pongaí',
+    'Pontal',
+    'Pontalinda',
+    'Pontes Gestal',
+    'Populina',
+    'Porangaba',
+    'Porto Feliz',
+    'Porto Ferreira',
+    'Potim',
+    'Potirendaba',
+    'Pracinha',
+    'Pradópolis',
+    'Praia Grande',
+    'Pratânia',
+    'Presidente Alves',
+    'Presidente Bernardes',
+    'Presidente Epitácio',
+    'Presidente Prudente',
+    'Presidente Venceslau',
+    'Promissão',
+    'Quadra',
+    'Quatá',
+    'Queiroz',
+    'Queluz',
+    'Quintana',
+    'Rafard',
+    'Rancharia',
+    'Redenção da Serra',
+    'Regente Feijó',
+    'Reginópolis',
+    'Registro',
+    'Restinga',
+    'Ribeira',
+    'Ribeirão Bonito',
+    'Ribeirão Branco',
+    'Ribeirão Corrente',
+    'Ribeirão do Sul',
+    'Ribeirão dos Índios',
+    'Ribeirão Grande',
+    'Ribeirão Pires',
+    'Ribeirão Preto',
+    'Rifaina',
+    'Rincão',
+    'Rinópolis',
+    'Rio Claro',
+    'Rio das Pedras',
+    'Rio Grande da Serra',
+    'Riolândia',
+    'Riversul',
+    'Rosana',
+    'Roseira',
+    'Rubiácea',
+    'Rubinéia',
+    'Sabino',
+    'Sagres',
+    'Sales',
+    'Sales Oliveira',
+    'Salesópolis',
+    'Salmourão',
+    'Saltinho',
+    'Salto',
+    'Salto de Pirapora',
+    'Salto Grande',
+    'Sandovalina',
+    'Santa Adélia',
+    'Santa Albertina',
+    "Santa Bárbara d'Oeste",
+    'Santa Branca',
+    "Santa Clara d'Oeste",
+    'Santa Cruz da Conceição',
+    'Santa Cruz da Esperança',
+    'Santa Cruz das Palmeiras',
+    'Santa Cruz do Rio Pardo',
+    'Santa Ernestina',
+    'Santa Fé do Sul',
+    'Santa Gertrudes',
+    'Santa Isabel',
+    'Santa Lúcia',
+    'Santa Maria da Serra',
+    'Santa Mercedes',
+    "Santa Rita d'Oeste",
+    'Santa Rita do Passa Quatro',
+    'Santa Rosa de Viterbo',
+    'Santa Salete',
+    'Santana da Ponte Pensa',
+    'Santana de Parnaíba',
+    'Santo Anastácio',
+    'Santo André',
+    'Santo Antônio da Alegria',
+    'Santo Antônio de Posse',
+    'Santo Antônio do Aracanguá',
+    'Santo Antônio do Jardim',
+    'Santo Antônio do Pinhal',
+    'Santo Expedito',
+    'Santópolis do Aguapeí',
+    'Santos',
+    'São Bento do Sapucaí',
+    'São Bernardo do Campo',
+    'São Caetano do Sul',
+    'São Carlos',
+    'São Francisco',
+    'São João da Boa Vista',
+    'São João das Duas Pontes',
+    'São João de Iracema',
+    "São João do Pau d'Alho",
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)